services/search.ts

```diff
@@ -161,6 +161,47 @@ export class SearchService extends Service {
     res.status(response.code).json(response)
   }
 
+  /**
+   * @description Performes a quick search for expertise, people and companies
+   * @param {express.Request} req
+   * @param {express.Response} res
+   */
+  public async query(req: express.Request, res: express.Response) {
+    let response: APIresponse = {
+      ok: 1,
+      code: 200,
+    }
+    const rawQuery = req.body.search
+    const query = rawQuery.replace(/\s/g, '').toLowerCase() // removing whitespaces and lowercase the query
+
+    try {
+      response.search = {}
+
+      const checkExistingQuery = `SELECT query FROM searchQuery WHERE query='${query}'`
```

Comment:
Please don't concatenate the user input into the SQL query. That leaves us open to SQL injection attacks. E.g. user input like `; DROP TABLE ...;`. 
The `query` has a second parameter that can be used to insert the user input into the query.

Reply:
yes, that makes sense. What do you mean with "The query has a second parameter that can be used to insert the user input into the query"?

Reply:
Oh, there was a word missing 😅
I mean the `query` function defined in `database.ts`. The second parameter takes a string or an array of SQL parameters. In the `quick` and `full` methods of `SearchService`, you can see it being used.